docs/ref/models/fields.txt

```diff
@@ -502,6 +502,15 @@ Any combination of these options will result in an error.
     ``True`` will cause the field to have ``editable=False`` and ``blank=True``
     set.
 
+.. note::
+    When using timezones, storing a ``DateField`` in the database is
+    ambiguous because it's incapable of storing timezone aware information, unlike ``DateTimeField``.
+    Unless you know you will need ``DateField``, consider using ``DateTimeField`` instead
+    and convert to a date format for display only.
```

Comment:
This advice doesn't sound good to me. The correct advice is "use the data type that matches the data you're modeling".

Datetimes are much more complicated than dates, especially when `USE_TZ = True`. I'm afraid users who naively follow this advice will hit issues they wouldn't have had if they had used dates.